main.tf

```
@@ -79,91 +50,45 @@ resource "aws_iam_role_policy_attachment" "vpc_attachment" {
   policy_arn = "arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole"
 }
 
-/*
-module "event-cloudwatch-scheduled-event" {
-  source = "./modules/event/cloudwatch-scheduled-event"
-  enable = lookup(var.event, "type", "") == "cloudwatch-scheduled-event" ? true : false
-  architecture = {}
-
-  lambda_function_arn = module.lambda.arn
-  schedule_expression = lookup(var.event, "schedule_expression", "")
-}
-*/
-
 module "lambda_cloudwatch_trigger" {
   source  = "app.terraform.io/bankrate/lambda-cloudwatch-trigger/aws"
   version = "~> 4.0.0"
   
   # Enablement
-  enable              = var.enable && var.architecture.cloudwatch_trigger
+  enable              = var.architecture.cloudwatch_trigger
```

I won't comment in a ton of places but we'll want to preserve the overarching "enable" flag to control whether the module creates resources at all for multi-env configs. Taking that a step further, lookup will allow them to pass in a struct that either has the key or doesn't, terraform will attempt to find the key "cloudwatch_trigger", if it finds it it will use the value (true or false) if it doesn't it will default to false.

So quick python example, the logic would be like:
```
enable = shouldInitModule()

...

def shouldInitModule( enable=False, architecture=None ):

    if enable and architecture is not None and "cloudwatch_trigger" in architecture:
        return architecture["cloudwatch_trigger"]

    return False 
```

In Terraform/HCL it'll look something like this:
```suggestion
  enable = var.enable && lookup(var.architecture, "cloudwatch_trigger", false) 
```